pkg/auth/rhsso_authenticator.go

```diff
@@ -162,8 +162,14 @@ func (a *RHSSOAuthenticator) AuthUserAuth(token string) (interface{}, error) {
 
 	// Check if there was an error in parsing...
 	if err != nil {
-		a.log.Errorf("Error parsing token: %s", err.Error())
-		return nil, errors.Errorf("Error parsing token: %v", err)
+		// Don't report error "Token used before issued"
+		// TODO: This validation is going to be removed in jwt-go v4, once jwt-go v4
+		// is released and we start using it, this validation-skip can be removed.
+		e, ok := err.(*jwt.ValidationError)
+		if !ok || (e.Errors&jwt.ValidationErrorIssuedAt == 0) {
```

Comment:
Can we create a test for this condition? It's not immediately clear to me that this works.
Or maybe create a helper method for determining if an error is a validation error and test that?

Reply:
Done.